skyportal/models.py

```diff
@@ -463,6 +476,25 @@ class Comment(Base):
 GroupComment = join_model("group_comments", Group, Comment)
 
 
+class Classification(Base):
+    classification = sa.Column(sa.String, nullable=False)
```

Comment:
I spent awhile on this idea previously. Turns out we cannot do that since we don't know the enum apriori. The enum depends on the taxonomy loaded. The `Taxonomy` handler now adds an `allowed_classes` list during a new post and the `Classification` handler ensures that the classification is allowed in that taxonomy.